### PR TITLE
Update help text styles

### LINKED ISSUE #1034: Build out help text style updates
Pulling this over from #962 — Update the help text styles to match this style:

image

To differentiate between instructions and help text, instructions should be details on how to actually fill out the section, "fill out XYZ" while help text should be additional contextual information that would help the user understand background or specific points to remember.

Implement a folded corner or triangle tab icon to a Light blue #DAE9F7 background instead of the yellow, which seems more like an alert or something wrong.

Be aware of:
- Remove bolding of any instructional text
- Be aware of what should go in what style in terms of instructive text vs useful context/help text.
- Not all sections or subsections will have both instruction text and help text, sometimes it's one or the other, which is ok.

Example

image

### PR DESCRIPTION
**Pull request by hbillings**
Closes #1034 and #1055.

### This pull request changes...
- alert styles now are the right colors and use the folded corner
- body copy throughout the form now uses `p` tags
- adds classes for font sizes/line heights `text-s`, `text-med`, and `text-xl`, with room for a potential `text-l` that may be necessary

### This pull request is ready to merge when...
- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author

### This feature is done when...
- [x] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Comment by hbillings** [comment]
Blocked by #1055 and #1076.

File: web/src/styles/app/alert.css [hunk 1]
```diff
@@ -1,25 +1,36 @@
...
+}
+
+.alert::before {
+  width: 0;
+  height: 0;
+  content: "";
+  display: block;
+  border-left: 12px solid transparent;
+  border-right: 12px solid transparent;
+  transform: translate(-2.4rem, -.95rem) rotate(-45deg);
```
Comment (mgwalker) [inline web/src/styles/app/alert.css]: I'd love to understand where these translation numbers come from  38.4px to the left and 15.2px up?  (If I understand correctly that 1rem is 16px, anyway.)  Is this to back out the left/top paddings plus some trigonometry due to the rotation?

File: web/src/components/Section.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ SectionTitle.propTypes = {
   children: PropTypes.node.isRequired
 };
 
-const SectionDesc = ({ children }) => <p className="mb4 h3">{children}</p>;
+const SectionDesc = ({ children }) => <p className="mb4 text-xl">{children}</p>;
```
Comment (mgwalker) [inline web/src/components/Section.js]: Oh come now, you don't want the whole paragraph to be a 3rd-level heading?

File: web/src/styles/app/alert.css (hunk 1 above)
Comment (hbillings) [reply web/src/styles/app/alert.css]: Yep! Although `trigonometry` in this case is actually `adjusting position a hundreth at a time with high contrast colors in dev tools`.

**Pull request merged**